src/content/blog/astro-570.mdx

```diff
@@ -0,0 +1,273 @@
+---
+title: 'Astro 5.7'
+description: 'Astro 5.7 has a basketload of treats, including stable Sessions and SVG components and a new Experimental Fonts API.'
+homepageLink:
+  title: 'Astro 5.7'
+  subtitle: 'Available now!'
+publishDate: '2025-04-15'
+authors:
+  - matt
+  - florian
+  - ema
+  - nate
+coverImage: '/src/content/blog/_images/astro-570/blog-post-5-7.webp'
+socialImage: '/src/content/blog/_images/astro-570/og-astro-5-7.webp'
+lang: 'en'
+---
+
+import BlogContentImage from '/src/components/BlogContentImage.astro';
+import Note from '~/components/Note.astro';
+import Mention from '/src/components/Mention.astro';
+
+**Astro 5.7 has a basketload of treats, including stable Sessions and SVG components and a new Experimental Fonts API.**
+
+🐣 Full of the joys of northern-hemisphere spring, Astro 5.7 is here with goodies galore:
+
+- [**Experimental Fonts API**](#experimental-fonts-api)
+- [**Sessions API**](#sessions-api)
+- [**SVG Components**](#svg-components)
+- [**Config Imports**](#config-imports)
+
+To upgrade an existing project, use the automated `@astrojs/upgrade` CLI tool. Alternatively, upgrade manually by running the upgrade command for your package manager:
+
+```sh
+# Recommended:
+npx @astrojs/upgrade
+
+# Manual:
+npm install astro@latest
+pnpm upgrade astro --latest
+yarn upgrade astro --latest
+```
+
+## Experimental Fonts API
+
+Almost every website these days use custom fonts. However, it can be surprisingly complicated to get them right: how many font faces should I include? Why do I get a Flash of Unstyled Text? Should I preload?
```

Comment:
> how many font faces should I include?

I'm not happy about this one but I feel like fonts are complicated except I can't find examples...

Reply:
I did a quick web search, and this article came up for "fonts best practices" -- is there something here that feels like something you'd point out that's helpful for justifying that fonts are tricky/have potentially performance effects on your site? https://web.dev/articles/font-best-practices

Reply:
Maybe we can talk about preconnect instead

```suggestion
Almost every website these days use custom fonts. However, it can be surprisingly complicated to get them right: why do I get a Flash of Unstyled Text? Should I add a preload link? What about preconnect links? Etc...
```